src/style/design-tokens/carbon-scoped-tokens-provider/carbon-scoped-tokens-provider.component.ts

```diff
@@ -9,7 +10,9 @@ import generateCssVariables from "../generate-css-variables.util";
  *
  */
 
-const CarbonScopedTokensProvider = styled.div`
+const CarbonScopedTokensProvider = styled.div<
+  ThemeObject | Record<string, string>
```

Comment:
Why are we giving the option to pass anything other than a `ThemeObject` into this?

Reply:
I think I must have had an incorrect type somewhere else, I'll get it removed 👍